app/src/main/java/nl/avans/drivioapp/viewModel/AddElectricCarViewModel.kt

```diff
@@ -0,0 +1,45 @@
+package nl.avans.drivioapp.viewModel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.launch
+import nl.avans.drivioapp.model.ElectricCar
+import nl.avans.drivioapp.service.AddElectricCarAPI
+import retrofit2.Response
+
+private const val TAG = "AddElectricCarViewModel"
+
+class AddElectricCarViewModel : ViewModel() {
+    private val _electricCarResponse: MutableLiveData<List<ElectricCar>> = MutableLiveData();
+    val electricCarResponse: LiveData<List<ElectricCar>>
+        get() = _electricCarResponse;
+
+    private val _getElectricCarByIdResponse: MutableLiveData<Response<ElectricCar>> = MutableLiveData()
+    val getElectricCarByIdResponse: LiveData<Response<ElectricCar>>
+        get() = _getElectricCarByIdResponse
```

Comment:
Elke call heeft zin eigen response nodig volgens mij. Anders krijg je een response te zien van een andere api call.

Reply:
Gebeurd in een nieuwe branch.